### PR TITLE
Update readme to provide more information about logging.properties.

### PR DESCRIPTION
**Pull request by chenrujun**
1. Add `logging.properties` to configure log
2. Update readme to provide more information about `logging.properties`.

### CODE REVIEW

**Comment by chenrujun** [comment]
The [pipeline failure] is not cause by this PR.
@vcolin7 , could you please help to fix the pipeline failure?

@yiliuTo , @michaelqi793 , could you please help to review this PR?



[pipeline failure]: https://dev.azure.com/azure-sdk/public/_build/results?buildId=1044185&view=logs&j=b70e5e73-bbb6-5567-0939-8415943fadb9&t=ba3cf43a-b2bf-50f9-f95f-93049ca44ad1&l=801

**Comment by vcolin7** [comment]
@chenrujun We probably updated RevApi and that's why it's complaining about something it didn't use to before. I can later make a PR to add an exception but for now I think it's safe to ignore the failure.

**Comment by chenrujun** [comment]
@vcolin7 . Thank you for your response. After your PR created, please share the link here.

File: sdk/keyvault/azure-security-test-keyvault-jca/src/test/resources/logging.properties [hunk 1]
```diff
@@ -0,0 +1,9 @@
+#  Add VM parameter to enable this configure file: -Djava.util.logging.config.file="src/test/resources/logging.properties"
+
+handlers = java.util.logging.ConsoleHandler
```
Comment (vcolin7) [inline sdk/keyvault/azure-security-test-keyvault-jca/src/test/resources/logging.properties:3]: We probably should add documentation on how to use this file either separately or via a few comments here. I don't know the difference between what the `java.util.logging.ConsoleHandler.level`, `.level` or `com.azure.security.keyvault.jca.level` properties do.

**Comment by vcolin7** [comment]
@chenrujun I want to wait to hear back from @srnagar and @alzimmermsft on [this comment](https://github.com/Azure/azure-sdk-for-java/pull/23523#issuecomment-897382387) before creating a PR, but I will definitely address the issue tomorrow depending on what they suggest I do. I don't think it's a problem to use `/check-enforcer override` here as the issue is not caused by your PR.

File: sdk/keyvault/azure-security-test-keyvault-jca/src/test/resources/logging.properties (hunk 1 above)
Comment (chenrujun) [reply sdk/keyvault/azure-security-test-keyvault-jca/src/test/resources/logging.properties]: Added a new commit to provide more information about this file.

File: sdk/keyvault/azure-security-keyvault-jca/README.md [hunk 2]
```diff
@@ -146,6 +146,33 @@ PEM          | EC       | P-521                           | SHA512withECDSA | 
 PEM          | EC       | P-256K                          |                 | ✘       |
 
 ## Troubleshooting
+
+## Configure logging
+This module use JUL(java.util.logging), so you can configure JUL to config the logging.
+
+Here is an example `logging.properties`:
```
Comment (vcolin7) [inline sdk/keyvault/azure-security-keyvault-jca/README.md:153]:
```suggestion
Here is an example of a `logging.properties` file:
```

File: sdk/keyvault/azure-security-keyvault-jca/README.md [hunk 3]
```diff
@@ -146,6 +146,33 @@ PEM          | EC       | P-521                           | SHA512withECDSA | 
...
+```properties
+# To enable this configure file, Please add this property:
```
Comment (vcolin7) [inline sdk/keyvault/azure-security-keyvault-jca/README.md:155]:
```suggestion
# To enable this configuration file, please add this property:
```

File: sdk/keyvault/azure-security-keyvault-jca/src/test/resources/logging.properties [hunk 4]
```diff
@@ -0,0 +1,18 @@
+# To enable this configure file, Please add this property:
```
Comment (vcolin7) [inline sdk/keyvault/azure-security-keyvault-jca/src/test/resources/logging.properties:1]:
```suggestion
# To enable this configuration file, please add this property:
```

File: sdk/keyvault/azure-security-keyvault-jca/src/test/resources/logging.properties [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+# 1. https://docs.oracle.com/javase/8/docs/technotes/guides/logging/overview.html#a1.8
+# 2. https://docs.oracle.com/cd/E23549_01/doc.1111/e14568/handler.htm
+
+handlers = java.util.logging.ConsoleHandler
+java.util.logging.ConsoleHandler.level = ALL
+java.util.logging.ConsoleHandler.formatter = java.util.logging.SimpleFormatter
+java.util.logging.SimpleFormatter.format= [%1$tF %1$tT] %3 [%4$-7s] %5$s %n
+
+.level = INFO
+com.azure.security.keyvault.jca.level = ALL
```
Comment (vcolin7) [inline sdk/keyvault/azure-security-keyvault-jca/src/test/resources/logging.properties:18]: What are these two for? I don't think it's fully explained in the Oracle documentation.

**Review by vcolin7 — COMMENTED** [review]
Added some more comments and suggestions.

**Comment by vcolin7** [comment]
@chenrujun I'm going to bed for now. I approved the PR to unblock you and think I left enough comments for anything that stood out. Thanks for taking care of this so quickly :)

File: sdk/keyvault/azure-security-keyvault-jca/src/test/resources/logging.properties (hunk 5 above)
Comment (chenrujun) [reply sdk/keyvault/azure-security-keyvault-jca/src/test/resources/logging.properties]: Just set the log level for this: package `com.azure.security.keyvault.jca.level`.

**Pull request merged**